Makefile

```diff
@@ -30,7 +30,7 @@ $(LOCAL_BINARY): $(SOURCES)
 
 .PHONY: test
 test:
-	. ./scripts/shared_env && env -i GO15VENDOREXPERIMENT=$$GO15VENDOREXPERIMENT PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT go test -timeout=120s -v -cover ./ecs-cli/license/... ./ecs-cli/modules/...
+	. ./scripts/shared_env && env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT go test -timeout=120s -v -cover ./ecs-cli/license/... ./ecs-cli/modules/...
```

Comment:
Do you still need to source `shared_env` here? I think you can get rid of it (and may be extract setting `PATH` if needed) as there's no need to set `GO15VENDOREXPERIMENT` there as well